src/main/java/cn/nukkit/block/Melon.java

```diff
@@ -0,0 +1,45 @@
+package cn.nukkit.block;
+
+import cn.nukkit.item.Item;
+import cn.nukkit.item.Tool;
+
+import java.util.Random;
+
+/*
+ * Created on 2015/12/11 by Pub4Game.
+ * Package package cn.nukkit.block in project Nukkit .
+*/
+
+public class Melon extends Transparent {
+
+    public Melon() {
+        this(0);
+    }
+
+    public Melon(int meta) {
+        super(meta);
```

Comment:
If you found codes like this: `public function __construct(){}` in PocketMine's block classes, use `super(0);` instead of `super(meta);`